src/components/EnterpriseList/__snapshots__/EnterpriseList.test.jsx.snap

```diff
@@ -396,35 +396,8 @@ Array [
                 placeholder="Search by enterprise name..."
                 role="searchbox"
                 type="text"
-                value="enterprise name"
+                value=""
               />
-              <button
```

Comment:
`CLARIFICATION`: not clear on this change, so the button that used to be there will no longer render?

Reply:
The `value` of the searchbox is now empty (line 399) which means no active search is applied. the button that is no longer rendering is the clear icon which makes sense if no query is applied.

that said, @raq929 this snapshot seems to check that it "renders correctly with search query and empty enterprises data" so I'm confused why the search query is now empty.

Reply:
Oooooh, yes I think this is actually broken. Will investigate, thanks.

Reply:
Update: Not actually broken when I test manually, but the snapshot sure it. =/

Reply:
Went with unit tests instead.